src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
Ah, different use cases. When you're typing out `bar`, it should the description. But after you've entered `bar: someValue` and hover over `bar` again, the description is lost.

Anyway, @darcyparker I agree - if you want to tackle it in a separate commit in this PR, feel free to get rid of single line comments in favor of multiline jsdoc-style comments for intellisense. Otherwise, feel free to do it in a separate PR, or just file an issue.